src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -302,7 +302,7 @@ async def handle_dispatch(
                         return
 
                     if event["type"] in [
-                        EVTYPE_ENSEMBLE_STOPPED,
+                        EVTYPE_ENSEMBLE_SUCCESS,
```

Comment:
Should we rename the EVTYPE_ENSEMBLE_SUCCESS to EVTYPE_ENSEMBLE_SUCCEEDED or the failed event type to EVTYPE_ENSEMBLE_FAILURE

Reply:
Changing to SUCCEEDED.

Reply:
Changing SUCCESS to SUCCEEDED.